doc/tasks/deptrac.md

```diff
@@ -0,0 +1,51 @@
+# Deptrac
+
+Follow the [installation instructions](https://github.com/sensiolabs-de/deptrac#installation) to add deptrac to your 
```

Comment:
Can't deptrac also be installed through composer?
Since it has a composer.json file in it's repository, I guess it should be possible.